setup.cfg

```diff
@@ -0,0 +1,45 @@
+# See https://packaging.python.org/en/latest/tutorials/packaging-projects
+
```

Comment:
nitpick: I'd remove this comment from here.

Reply:
I want to leave pointers to the documentation, it is the best explanation of what is going on.